src/settings.c

```diff
@@ -108,56 +101,17 @@ static int parse_args(int argc, char *argv[], struct settings *settings)
 struct settings *settings_load(int argc, char *argv[])
 {
 	struct settings *settings;
-	struct stat buf;
```

Comment:
Since we don't use this struct anymore we can remove header `sys/stat.h`

Reply:
Ok